react/PWAContext.js

```diff
@@ -54,10 +54,12 @@ const PWAProvider = ({ rootPath, children, data = {} }) => {
   const context = useMemo(() => {
     if (pwaSettings) {
       const { disablePrompt, promptOnCustomEvent } = pwaSettings
+      const isIOS = window && window.navigator && !!window.navigator.userAgent.match(/(iPod|iPhone|iPad)/)
```

Comment:
could you use `navigator` directly?

```suggestion
      const isIOS = navigator && !!navigator.userAgent.match(/(iPod|iPhone|iPad)/)
```